src/items/external-blocks.md

```diff
@@ -17,7 +17,7 @@
 >
 > _ExternalFunctionItem_ :\
 > &nbsp;&nbsp; `fn` [IDENTIFIER]&nbsp;[_Generics_]<sup>?</sup>\
-> &nbsp;&nbsp; `(` _NamedFunctionParameters_<sup>?</sup> | _NamedFunctionParametersWithVariadics_ ) `)`\
+> &nbsp;&nbsp; `(` ( _NamedFunctionParameters_<sup>?</sup> | _NamedFunctionParametersWithVariadics_ ) `)`\
```

Comment:
Thanks! I think it might be a little clearer to move the <sup>?</sup> outside the parenthesis.  I vaguely recall messing with this, and might have been what was intended.

```suggestion
> &nbsp;&nbsp; `(` ( _NamedFunctionParameters_ | _NamedFunctionParametersWithVariadics_ )<sup>?</sup> `)`\
```